### DISCUSSION TITLE
First-party, in-game skin library.

### DISCUSSION THREAD

**Discussion by GOINGCRAZYWITHIT** [ppy/osu#27150] [Ideas] [answered]
Because of the ability to edit skins in game with presumed additions and improvements to it in the future, a place for skin creators to upload their skins and players to upload their own positionings of the skins sounds very useful. Such a feature, especially polished, is definitely far out, but feels like a logical continuation of the extensive skin editor.

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/17759. Please upvote that one to show your support!
